seisidd/config/scan_template.yml

```diff
@@ -0,0 +1,103 @@
+# NOTE:
+# This is a yaml based scan configuration file that can be directly pass
+# to the tomo_scan() function or nf_scan(), ff_scan()
+# i.e.
+# RE(tomo_scan(SCAN_CONFIG.YML))
+
+# Will use a centralized yml for all the scans, notebook will spit out individual scan configs/pdf
+
+
+
+tomo:
+  ## Beamline optics
+  energyfoil:   "Re"      #   Ho, Pb, Re, Tb, Air, Au, Yb, Bi, Pt, Ir, Hf, Tm, Ta
+                          #   probably just need to do once???
+  attenuation:  "a number"         # Place hodler. This doesn't seem to be pre-defined for the entire run
+  FS_control:   "sweep"   # sweep, detpulse, detexposure etc. used this to determine FS control
+  beamsize_h:   0.5       # horizontal beam size
```

Comment:
The beam size is from the slit motor set up, which is in mm (typically). I don't think we should use pixels, because that would depend on the actual detector used.  The physical dimensions of the slits would make more sense as this relates to the sampling size.